pkg/k8s/watchers/watcher.go

```diff
@@ -644,13 +644,17 @@ func genCartesianProduct(
 			parsedIP := net.ParseIP(netIP)
 
 			if backendPort := backend.Ports[string(fePortName)]; backendPort != nil && feFamilyIPv6 == ip.IsIPv6(parsedIP) {
+				backendState := loadbalancer.BackendStateActive
+				if backend.Terminating {
```

Comment:
This `backend.Terminating` comes from k8s api, right?

Propagation to BPF map and restoration is still on TODO, right?

Reply:
Synced up offline.